contracts/okp4-cognitarium/src/msg.rs

```diff
@@ -28,6 +28,42 @@ pub enum ExecuteMsg {
         /// are subject to the limitations defined by the `limits` specified at contract instantiation.
         data: Binary,
     },
+
+    /// # DeleteData
+    /// Delete the data (RDF triples) from the store matching the patterns defined by the provided
+    /// query. For non-existing triples it acts as no-op.
+    ///
+    /// Example:
+    /// ```json
+    /// {
+    ///   "prefixes": [
+    ///     { "prefix": "foaf", "namespace": "http://xmlns.com/foaf/0.1/" }
+    ///   ],
+    ///   "delete": [
+    ///     { "variable": "person" },
+    ///     { "variable": "property" },
+    ///     { "variable": "value" }
```

Comment:
I don't understand the presence of the `property` and `value` variables in the example, is that an error?

Reply:
yes, well seen. It's a typo. Initially, I started by providing an example with 3 variables, but then reduced to a single variable. I will fix.